mongoimport/mongoimport.go

```diff
@@ -27,6 +27,13 @@ const (
 	JSON = "json"
 )
 
+// Modes accepted by mongoimport.
+const (
+	MODE_INSERT = "insert"
```

Comment:
Standard constants like this should be formatted in camel case to follow the convention in Go's libraries. (e.g. `modeInsert = ...`)